.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,7 +5,7 @@ about: Create a report to help us improve
 
 * .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)
 * Windows version: (`winver`)
-* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes/No
+* Does the bug reproduce also in WPF for .NET Framework 4.8 on Windows?: Yes/No
```

Comment:
@cdesewell thanks, but I don't quite see how it adds more clarity.

The previous line already hints Windows-only (asking for Windows version / `winver`).
Moreover, .NET Framework 4.8 is only on Windows, so I am not sure that this will help ...

Note that [README.md](https://github.com/dotnet/wpf/blob/master/README.md#windows-presentation-framework-wpf) also clarifies we are Windows only at this point.

If we see continuous stream of misunderstandings from contributors past today, I would be ok with making some changes to the template clarifying Windows-only even more ... let's wait for a few such reports.
What do you think?

Reply:
@karelz I do feel we need a clearer way for dependencies and nuget packages to clearly state "Windows only" which would generate a very helpful obvious error message

Reply:
@dotMorten do you mean in general? Or is it specific to WPF?

Reply:
Anything relying on the Desktop SDK. I'd also prefer that nuget packages instantly display "sure this is .NET Core 3.0 compatible, but beware that it'll only work on windows desktop"

Reply:
@terrajobst any thoughts on that? We have the same problem already with some Windows Compat Pack packages.

Reply:
This might be a bigger issue wrt to nuget that should be discussed elsewhere - being able to limit packages and frameworks to certain runtimes, and have those dependencies become viral up the dependency stack